build.rs

```diff
@@ -78,7 +78,7 @@ fn main() {
     let silver_status = setup_namespace(
         curr_dir,
         "refs/namespaces/golden/refs/namespaces/silver/refs/heads/master",
-        "refs/remotes/origin/dev",
```

Comment:
When cargo grabs surf as a dependency it doesn't have a notion of origin as a remote. But we do have `refs/heads/dev` so we use that instead.